### DISCUSSION TITLE
Scrutini secondo pentamestre.

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#717] [Richieste di aiuto] [answered]
Buon pomeriggio, è possibile già svolgere gli scrutini delle classi per il secondo pentamestre oppure ci sarà un apposito aggiornamento per il prossimo scrutinio?

**Comment by trinko** [comment, marked as answer]
Si, gli scrutini finali sono stati testati. Esegui anche gli ultimi aggiornamenti inseriti dopo l'uscita della 1.5.8, anche se sono correzioni su problemi secondari.
